coderd/coderd.go

```diff
@@ -1281,6 +1281,7 @@ func New(options *Options) *API {
 			r.Use(apiKeyMiddleware)
 			r.Get("/daus", api.deploymentDAUs)
 			r.Get("/user-activity", api.insightsUserActivity)
+			r.Get("/user-status-counts-over-time", api.insightsUserStatusCountsOverTime)
```

Comment:
Suggestion: This seems a bit long, could we simplify to just `user-status-counts`?

Reply:
done.